src/plugins/filter/base.scss

```diff
@@ -1,3 +1,28 @@
 .wb-fltr-out {
 	display: none !important;
 }
+
+//Screen size large/x-large
+.wb-filter {
+	.input-group {
+		max-width: 60%;
+	}
+}
+
+//Screen size medium
+@media screen and (max-width: 960px) {
```

Comment:
Look at the Tabs plugin for how to deal with media queries in a plugin.

Also, your SASS file name should start with an underscore.